start.sh

```diff
@@ -250,6 +250,11 @@ function parse_compose_additional() {
     done
 }
 
+function wait_for_services() {
+  echo "Waiting for services to start..."
+  sleep 5
+}
+
```

Comment:
Please remove it, as it seems to be not used, and I don't see any reason to actually have this function